mitmproxy/net/http/request.py

```diff
@@ -82,8 +82,8 @@ def make(
             cls,
             method: str,
             url: str,
-            content: AnyStr = b"",
-            headers: Union[Dict[AnyStr, AnyStr], Iterable[Tuple[bytes, bytes]]] = ()
+            content: bytes = b"",
```

Comment:
`content` can be `str` here, see https://github.com/mitmproxy/mitmproxy/pull/2169/files#diff-477e6d69234066ab1a95fcfa034f295cR119.